templates/login.html

```diff
@@ -32,10 +32,10 @@
 
 <div class="container login-header" id="home" data-authenticator-login-url="{{authenticator_login_url}}">
   <div class="col-md-12 text-center">
-    <img src="{{static_url("extra-assets/images/logos/epa.svg") }}" alt='Jupyter' height="64" />
-    <img src="{{static_url("extra-assets/images/logos/nasa.png") }}" alt='Jupyter' height="64" />
-    <img src="{{static_url("extra-assets/images/logos/nist.png") }}" alt='Jupyter' height="64" />
-    <img src="{{static_url("extra-assets/images/logos/noaa.png") }}" alt='Jupyter' height="64" />
+    <img src="{{static_url("extra-assets/images/logos/epa.svg") }}" alt='EPA logo' height="64" />
+    <img src="{{static_url("extra-assets/images/logos/nasa.png") }}" alt='NASA logo' height="64" />
+    <img src="{{static_url("extra-assets/images/logos/nist.png") }}" alt='NIST logo' height="64" />
+    <img src="{{static_url("extra-assets/images/logos/noaa.png") }}" alt='NOAA logo' height="64" />
```

Comment:
Changed alt text to accurately describe the logos and provide a better experience for screen readers.